### ISSUE TITLE
60 Second City Hall Lobby Video

### ISSUE THREAD

**Issue by DACole** [CityOfBoston/av-club#19]
Once the the new entrance is finished being installed, use the old video of the space to show the change from what it was to what it is now. This can be filmed just after close, so that the images can be clear of people to see the detail in order. 

A capture during the day with the public present could also be interesting as well.

**Comment by DACole** [comment]
We filmed the initial state of the lobby during the CYF2017 with the space empty on the ronin.

**Comment by DACole** [comment]
footage was backed up locally.

**Comment by DACole** [comment]
awaiting finish of the build.

**Comment by DACole** [comment]
After the final build was done, we filmed some more shots that could be added to the before construction footage. After further review this project has been put on hold, and more content maybe added to the video.
